packages/components/src/themes/codesandbox-black.ts

```diff
@@ -129,13 +131,13 @@ const colors = {
     prominentHoverBackground: tokens.grays[600],
   },
   input: {
-    background: tokens.grays[600],
-    foreground: tokens.white,
-    border: tokens.grays[500],
-    placeholderForeground: tokens.grays[300],
+    background: tokensV2.black[500],
+    foreground: tokensV2.gray[200],
+    border: null,
```

Comment:
I'm not 100% sure what this will do and if it will maybe have the value `null` as an actual border color. Does anyone know if we have an input somewhere that uses this token? I should try storybook maybe 🤔.

Reply:
@tristandubbeld I believe it's parsed into an empty string. Seems to be used by the `AddDependency` modal opener button and the checkout form (which I don't think it's used anymore).

Reply:
@lenasmayor do you know what this change of border color has affected?

Reply:
we had some instances of using null as a border color on this very file `codesandbox-black.ts`. I just went with the flow, but I'm not sure if there are any unforeseen implications of going with this.

Reply:
we had some instances of using null as a border color on this very file `codesandbox-black.ts`. I just went with the flow, but I'm not sure if there are any unforeseen implications of going with this.